### PR TITLE
Updates README and adds top level GoDoc blurb

### LINKED ISSUE #264: Update readme with packages
The README should be updated to include a blurb about:
- paketosbom (deprecated)
- sbom
- servicebindings

### PR DESCRIPTION
**Pull request by ForestEckhardt**
Resolves #264 
## Checklist

* [x] I have viewed, signed, and submitted the Contributor License Agreement.
* [x] I have linked issue(s) that this PR should close using keywords or the Github UI (See [docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue))
* [ ] I have added an integration test, if necessary.
* [x] I have reviewed the [styleguide](https://github.com/paketo-buildpacks/community/blob/main/STYLEGUIDE.md) for guidance on my code quality.
* [x] I'm happy with the commit history on this PR (I have rebased/squashed as needed).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # packit
 
-[![GoDoc](https://img.shields.io/badge/pkg.go.dev-doc-blue)](http://pkg.go.dev/github.com/paketo-buildpacks/packit/v2)
+[![GoDoc](https://img.shields.io/badge/pkg.go.dev-doc-blue)](http://pkg.go.dev/.)
```
Comment (ryanmoran) [inline README.md]: Do we really want to remove this?

File: README.md [hunk 2]
```diff
@@ -213,17 +213,19 @@ the types and functions declared herein.
...
 * [matchers](./matchers)
 
+* [paketosbom](./paketosbom): Package paketosbom implements a standardized SBoM format that can be used in Paketo Buildpacks.
+
 * [pexec](./pexec): Package pexec provides a mechanism for invoking a program executable with a varying set of arguments.
 
 * [postal](./postal): Package postal provides a service for resolving and installing dependencies for a buildpack.
 
-* [scribe](./scribe)
+* [sbom](./sbom)
```
Comment (ryanmoran) [inline README.md]: Should we have the same blurbs you added for `sbom` and `servicebindings` packages in these locations?

Comment (ForestEckhardt) [reply README.md]: Yup don't know how those didn't make it in. It should be up-to-date now.

File: README.md (hunk 1 above)
Comment (ForestEckhardt) [reply README.md]: Yeah this was a miss too. It has also been updated.

**Review by ryanmoran — APPROVED** [review]

**Pull request merged**